src/handlers/delegatedStakingHandler.ts

```diff
@@ -0,0 +1,192 @@
+import { PrismaClient } from "@prisma/client";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { extractHashOrdinal, extractPagination } from "../request-params";
+import {
+  paginatedQuery,
+  fromCreatedAtOrdinalCursor,
+  toCreatedAtOrdinalCursor,
+  toOrdinalCursor,
+  fromOrdinalCursor,
+} from "../pagination";
+import { handleError, respond } from "../response";
+import { toNumber } from "lodash";
+import { latestGlobalSnapshot } from "./dagHandler";
+
+const prisma = new PrismaClient();
+
+const delegateStakeCreateResponse = (event) => ({
+  hash: event.hash,
+  ordinal: event.ordinal,
+  source: event.source_addr,
+  nodeId: event.node_id,
+  amount: event.amount,
+  fee: event.fee,
+  tokenLockRef: event.token_lock_ref,
```

Comment:
We use "hash" everywhere else. Should we call this `tokenLockHash`?

Reply:
yeah, I like it more.  use Hash instead of Ref